docs/state.md

```diff
@@ -21,6 +21,14 @@ AppStore
     startTime: number, // datetime.getTime()
     endTime: number // datetime.getTime()
   }],
+  passwords: [{
```

Comment:
Still undecided whether I want to make this change. The benefits of having it the way it is now are:
1. There are some operations that can be done without getting the master password, like seeing which passwords have been saved and deleting them once we have a password manager settings page.
2. No need to figure out `encryptedData` serialization/deserialization.

I think (1) is reason enough to keep it as is for now.